pygenometracks/tracks/HiCMatrixTrack.py

```diff
@@ -98,11 +100,15 @@ def set_properties_defaults(self):
                 region = [str(self.properties['region'][0]) + ':' + str(start) + '-' + str(end)]
         # try to open with end region + depth to avoid triangle effect in the plot
         # if it fails open it with given end region.
+        # if it still fails do not specify the region.
         try:
             self.hic_ma = HiCMatrix.hiCMatrix(self.properties['file'], pChrnameList=region)
         except Exception:
             region = [str(self.properties['region'][0]) + ':' + str(start) + '-' + str(self.properties['region'][2])]
-            self.hic_ma = HiCMatrix.hiCMatrix(self.properties['file'], pChrnameList=region)
+            try:
```

Comment:
I s this necessary?

Reply:
@joachimwolff can you confirm this please? I don't think that this try/exception is needed here. 
Does having or not having `pChrnameList` matter?

Reply:
I thought that if you would give chrX instead of X it would not work but it worked, so you are right, I will remove this.

Reply:
What I experienced is that if you give a list which has a length of 2 this exit.

Reply:
From the code, it looks like it could fail:
https://github.com/deeptools/HiCMatrix/blob/cd54a2e7982bc0880e17536a44b09459620cb6db/hicmatrix/lib/cool.py#L106-L120
but I did not manage to make it fail maybe this is linked to the version of cooler.

Reply:
doesn't it have something to do with you setting the the default value to none? (` DEFAULTS_PROPERTIES = {'region': None`) Isn't region supposed to be a list?

Reply:
I see how you mean. I think it means that it can only load one region at the time, this can be confirmed by @joachimwolff . Maybe if you have more than one region you need to it recursively ?

Reply:
I managed:
If the chromosome in the cool is UCSC style (chr1, chr2...) and you want to plot `--region Y:2500000-2600000` you get:
`Wrong chromosome format. Please check UCSC / ensembl notation.` but as it is an `exit` you cannot except it...

Reply:
So, I will remove the try/catch and check it is UCSC format as UCSC format works on ensembl format but not the contrary.
@joachimwolff, in a next release, could you transform all the exit in exception, so we could handle them?

Reply:
I think it was not a bad idea fail or at least warn the user about format. just adding chr at the beginning of every single chr can make troubles

Reply:
I think we really need to be able to catch the errors, @joachimwolff do you want me to do a PR?
I realized that if the region to plot is the end of the chromosome, then it will exit and we have no way to deal with it (and this but in independent of this PR this is a current bug).
If you want to deal with matrices in both format (ucsc and ensembl) it would be good to be able to plot both...

My prefered solution would be:
1. update HiCMatrix to be able to catch the error and even better, check that the pChromList does not go over the chromosome size.
2. use try/Catch to deal with ucsc/ensembl and if it is not implemented in HiCMatrix, to deal with region to get matrix above the chr size.
3. have a good conversion UCSC/Ensembl because we will still have issues with contigs.